std/types/spend_bundle.py

```diff
@@ -23,6 +23,10 @@ class SpendBundle(Streamable):
     coin_solutions: List[CoinSolution]
     aggregated_signature: G2Element
 
+    @classmethod
+    def from_chia_spend_bundle(cls, bundle):
```

Comment:
Why is this necessary?

Reply:
Might not be :-)

Reply:
added comments about this, and sign_coin_solutions, basically sign_coin_solutions from the main code is flexible enough to handle signing nonstandard spends, which the naive way of using AugSchemeMPL's sign method isn't (at least i'm unsure how to form it, so added the other on adam's advice), since that yields a chia style spend bundle, we need to convert it, but i've moved that internal to the exposed synchronous method that does signing, so it isn't exposed outside that system.